invenio_communities/views/ui.py

```diff
@@ -192,6 +200,7 @@ def register_menus():
             "invenio_communities.communities_new",
             _("New community"),
             order=3,
+            visible_when=_can_create_community,
```

Comment:
I see, my initial approach was to keep the default state visible to avoid disrupting the Zenodo setup with additional configurations. However, your suggestion entails the need to introduce a new configuration. I will adjust my updates accordingly. 👍🔄